resources/js/Components/Alert.vue

```diff
@@ -0,0 +1,19 @@
+<template>
```

Comment:
Not sure if we should commit this, maybe we should include an even simpler component? Something like our website Title or the small svg + log in status link that we have in the blade views?

Reply:
Addressed. Good idea!